### ISSUE TITLE
Disable URL preview

### ISSUE THREAD

**Issue by KaKi87** [nextcloud/notes#1033]
**Is your feature request related to a problem? Please describe.**
The app always shows URL previews in files.

**Describe the solution you'd like**
The app would allow disabling the preview by :
- surrounding the URL with `<` and `>` ;
- clicking a close button on the preview element ;
- changing a file-specific setting ;
- changing a global setting.

**Describe alternatives you've considered**
None

**Additional context**
![Screenshot from 2023-05-05 08-08-04](https://user-images.githubusercontent.com/21284089/236387658-20ac10ac-1aa1-4b3a-b1c6-c7cb04472d62.png)

---
Thanks

**Comment by babolivier** [comment]
I strongly agree that there should be a way to toggle URL preview on and off. I sometimes use notes to store lists of stuff that include URLs, and URL previews render the app unusable for this use case due to the height they take.

One way to disable that is to include text that isn't a URL on the same line as the URL, but it feels awkward having to add extra text I don't need just to work around a feature.

**Comment by DKX47** [comment]
> One way to disable that is to include text that isn't a URL on the same line as the URL, but it feels awkward having to add extra text I don't need just to work around a feature.

Thanks for the Trick. But I agree, it doesn't feel right to add random text in front or add "Link:" before every link

**Comment by alexboss** [comment]
I second this enhancement, but mainly for privacy reasons.
When I add a link to a note, I don't want to spread the IP address nor the domain name of my cloud to the destination link that will be pinged / tracked-back by Nextcloud Notes.
Thanks and best regards.

**Comment by kartoffelheinz** [comment]
This "feature" is so unbelievably annoying and a huge privacy and security risk. 
There must be an option to disable it completely for any given whole instance ASAP.

**Comment by evilJazz** [comment]
This should definitely be configurable in config.php or ideally in the admin settings.
'has_internet_connection' => false in config.php is ignored.

This is a major privacy leak/issue concerning GDPR/DSGVO.

I just came across this issue while auditing my Nextcloud docker containers with OpenSnitch and seeing that it opened external connections to seemingly random servers to resolve previews of the URLs. Bad design.

**Comment by alexanderdd** [comment]
since Nextcloud 29, when you insert a link, no preview is shown by default. But you can activate the preview via the ...-menu on the left.

@korelstar so this can be closed, right?
